### PR TITLE
fix some typo and create utils

### PR DESCRIPTION
**Pull request by bxdoan**
Hi @pvolnov, just update some typos, create constants and create utils to centralize helper

### CODE REVIEW

File: src/pynear/account.py [hunk 1]
```diff
@@ -8,6 +7,8 @@
 from pyonear.crypto import InMemorySigner, ED25519SecretKey
 from pyonear.transaction import Action
 
+import pynear.utils as Utils
```
Comment (pvolnov) [inline src/pynear/account.py:10]: why capitalize it? 

```
from pynear import utils

utils.timestamp()
```

**Review by pvolnov — CHANGES_REQUESTED** [review]

File: src/pynear/account.py (hunk 1 above)
Comment (bxdoan) [reply src/pynear/account.py:10]: It's just my style code, do you want me to update for constants also?

`import pynear.constants as Constants`

Comment (pvolnov) [reply src/pynear/account.py]: yes

**Comment by bxdoan** [comment]
Hi @pvolnov , I have updated it, please check again

**Review by pvolnov — APPROVED** [review]

**Pull request merged**